src/actions/testActions.ts

```diff
@@ -88,6 +86,11 @@ const setupTest = async ({
     modulationMode: number;
     phy: number;
 }) => {
+    const dtm = getDTM();
+    if (!dtm) {
+        throw new Error('DTM object is undefined');
+    }
+
```

Comment:
Nit I guess, but why not just pass it as an argument

Reply:
We can originally that was the idea then had to change (as I was calling tis in the UI where I only had serial port or baud rate .... but we can yet .... this is a temp fix for I think we can le tit be for now